jobs/views.py

```diff
@@ -0,0 +1,14 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+
+from .models import Jobs
+
+
+# Create your views here.
+def job_view(request, *args, **kwargs):
+    # get 20 records from database
+    jobs = Jobs.objects.all()[:20]
```

Comment:
Order it in descending order, so latest are first

Reply:
done